clouddriver-cloudfoundry/src/test/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/converters/DeployCloudFoundryServiceAtomicOperationConverterTest.java

```diff
@@ -176,8 +180,42 @@ void convertDescriptionWithDownloadedManifest() {
         .setServicePlan("my-service-plan")
         .setTags(Collections.singleton("tag1"))
         .setParameterMap(HashMap.<String, Object>of(
-        "foo", "bar"
-      ).toJavaMap())
+          "foo", "bar"
+        ).toJavaMap())
+    );
+  }
+
+  @Test
+  void convertDescriptionWithUserProvidedInput() {
```

Comment:
What does this test other than basic Jackson deserialization?

Reply:
The intention was that it would deserialize into `UserProvidedServiceAttributes` and not `ServiceAttributes`. I've Added an an additional assertion that it would not deserialize the object to `ServiceAttributes` to make that more clear.